### PR TITLE
Docs: Add subnets to the left nav of the documentation

### PR DESCRIPTION
**Pull request by LakshmiLavanyaKasturi**
Added getting started guide for Subnets. 

Files changed:

- sidebar.js
- subnets/getting-started.md
- subnets/overview.md

Changed introduction -> overview in the Clarinet files to keep the product consistency.

### CODE REVIEW

File: docs/subnets/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+# The default id is the same as the one being defined below. so not needed
+title: Getting Started
+---
+
+# Getting started
+
+You can get started building subnets using the [subnets-readme](https://github.com/hirosystems/stacks-subnets/blob/master/README.md) documentation.
```
[the remark this answers is not in the document]
Comment (LakshmiLavanyaKasturi) [reply docs/subnets/getting-started.md:8]: Had mixed thoughts about it until I pushed the PR. :-) I guess it makes sense to keep the consistency until we solve the duplicating problem. I will change it.

[the remark this answers is not in the document]
Comment (LakshmiLavanyaKasturi) [reply docs/subnets/getting-started.md]: Changes are pushed to the PR. @sabbyanandan and @trichards-hiro. Please review. Thanks!

**Review by kantai — CHANGES_REQUESTED** [review]
The `overview.md`, `trust-models.md`, `getting-started.md` content looks fine.

The `how-to-guides`, however, I think don't make sense here: each of these how-to's is actually just a couple steps from the [NFT_USE_CASE.md](https://github.com/hirosystems/stacks-subnets/blob/master/NFT_USE_CASE.md), and the steps all depend on each other, so the how-to's don't make sense by themselves. This could be solved by just making this part of the docs a "NFT use case walkthrough" or similar. Otherwise, the docs will need to be updated so that these things make more sense individually.

But at a high level, the subnets repo is still under active development, and some of this content is likely to need to change. This content all looks directly derived from the MD files in the subnets repo, is it possible to build these doc files from that repo? Or, just ensure that the files are actually identical, so that any updates later are easier?

**Comment by LakshmiLavanyaKasturi** [comment]
> The `overview.md`, `trust-models.md`, `getting-started.md` content looks good to me.
> 
> The `how-to-guides`, however, I think don't make sense here: each of these how-to's is actually just a couple steps from the [NFT_USE_CASE.md](https://github.com/hirosystems/stacks-subnets/blob/master/NFT_USE_CASE.md), and the steps all depend on each other, so the how-to's don't make sense by themselves. This could be solved by just making this part of the docs a "NFT use case walkthrough" or similar. Otherwise, the docs will need to be updated so that these things make more sense individually.
> 
> But at a high level, the subnets repo is still under active development, and some of this content is likely to need to change. This content all looks directly derived from the MD files in the subnets repo, is it possible to build these doc files from that repo? Or, just ensure that the files are actually identical, so that any updates later are easier?

Thanks for your quick response. I will put together all the how-to's into one guide. 
We recently pushed a plugin that pulls remote content to docs. So, for your question yes we do have the option to pull the content from the Subnet repository to docs without duplication.

File: docs/subnets/feature-guides/trust-models.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+title: Trust Models
+---
+
+
+The current implementation of subnets uses a federated system of miners. This federation is fully-trusted, but future work on subnets will explore alternative trust models.
+
+In a fully - trusted model:
```
Comment (saralab) [inline docs/subnets/feature-guides/trust-models.md:8]: In "this fully trusted federated model"

**Comment by LakshmiLavanyaKasturi** [comment]
Closing this PR as @trichards-hiro will come up with a new PR.

**Pull request closed without merging**